domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/GjeldendeVedtaksdata.kt

```diff
@@ -1,25 +1,28 @@
 package no.nav.su.se.bakover.domain.revurdering
 
+import arrow.core.NonEmptyList
 import no.nav.su.se.bakover.common.Tidspunkt
+import no.nav.su.se.bakover.common.between
 import no.nav.su.se.bakover.common.periode.Periode
 import no.nav.su.se.bakover.domain.grunnlag.Grunnlag
 import no.nav.su.se.bakover.domain.grunnlag.Grunnlagsdata
-import no.nav.su.se.bakover.domain.vedtak.Vedtak
+import no.nav.su.se.bakover.domain.vedtak.VedtakSomKanRevurderes
 import no.nav.su.se.bakover.domain.vedtak.lagTidslinje
 import no.nav.su.se.bakover.domain.vedtak.vilkårsvurderinger
 import no.nav.su.se.bakover.domain.vilkår.Vilkår
 import no.nav.su.se.bakover.domain.vilkår.Vilkårsvurderinger
+import java.time.LocalDate
 import java.util.UUID
 
-data class KopierGjeldendeGrunnlagsdataOgVilkårsvurderinger(
-    private val periode: Periode,
-    private val vedtakListe: List<Vedtak>,
+data class GjeldendeVedtaksdata(
+    val periode: Periode,
```

Comment:
Hva med å kalled denne for revurderingsperiode? Eller endringsperiode? Eller tenker vi at den kan være noe annet?

Reply:
I utgangspunktet kunne jeg tenkt meg at funksjonaliteten var helt uavhengig av konseptet revurdering, da funksjonaliteten i seg selv har andre bruksområder enn bare det. En liten mismatch her er at klassen kun aksepterer `VedtakSomKanRevurderes`, som for så vidt bare er et navn som kunne vært endret til noe annet (gav mening i den contexten hodet mitt var i da jeg navnga det). Forslag?

Reply:
Eneste forslaget jeg har er VedtakSomKanEndres (men det høres jo litt rart å kunne endre et vedtak), men de bruker vel det generelle ordet endring i nav? På tvers av revurdering, klage etc.

Reply:
Flytter vel kanskje forvirringen til om et vedtak faktisk kan endres eller ikke?